### ISSUE TITLE
How do you use Validation data when training MTER?

### ISSUE THREAD

**Issue by lthoang** [MyTHWN/MTER#5]
Hi Nan Wang,

As described in your paper, the split ratio for train/validation/test is 80/10/10.
Is validation data only used for optimal hyper parameters search for baseline models?
For MTER, do you use validation set for model selection purpose?

Thanks,
Hoang

**Comment by MyTHWN** [comment]
Yes, the validation data is used to tune hyper-parameters for all models.

**Comment by lthoang** [comment]
Alright. I have a follow up question.
What is the criteria that you used to select the best model for MTER? Best loss or best NDCG on validation set?

**Comment by MyTHWN** [comment]
Best NDCG
